src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--thu calendar--month-length--29">
```

Comment:
[FUNCTIONALITY] - Your project should work correctly (have 31 days and start on Monday) with invalid modifier values, like start-day--ghy or days--27. 
I think you should remove modifier

Reply:
Everything is fine here